File: existant/gestFormationWeb/gestion_formation_web/dto/DtoCursus.cs
using System;
using System.Collections.Generic;

using gestion_formation_web.dao;

namespace gestion_formation_web.dto
{
    public class DtoCursus
    {
        public static IEnumerable<cursus> Get()
        {
            return DaoCursus.Get();
        }
        public static cursus Get(int idCursus)
        {
            return DaoCursus.Get(idCursus);
        }
        public static IEnumerable<session_cursus> GetSessionsCursus(int idCursus)
        {
            return DaoCursus.GetSessionsCursus(idCursus);
        }

        public static IEnumerable<formation> GetFormations(int idCursus)
        {
            return DaoCursus.GetFormations(idCursus);
        }

        public static IEnumerable<formation> GetAutresFormations(int idCursus)
        {
            return DaoCursus.GetAutresFormations(idCursus);
        }

        internal static void Add(int idCursus, int idFormation)
        {
            //cursus leCursus = Get(idCursus);
            //formation laFormation = DtoFormation.Get(idFormation);
            //DaoCursus.Add(leCursus, laFormation);
             DaoCursus.Add(idCursus, idFormation);
        }
    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoCursus.java
package fr.esic.solutec.dto;

import java.util.List;

import fr.esic.solutec.dao.DaoCursus;
import fr.esic.solutec.entities.Cursus;

public class DtoCursus {

	
	public static Cursus getCursus(int idCursus) {
		
		return DaoCursus.getCursus(idCursus);
	}
	
	public static List<Cursus> getListCursus() {
		
		return DaoCursus.getListCursus();
	}
	
	
	public static int AddCursus(Cursus cursus) {

		return DaoCursus.AddCursus(cursus);
	}
	
	public static void EditCursus(Cursus cursus) {
		DaoCursus.EditCursus(cursus);
	}
	
}
